src/js/components/NotesView.js

```diff
@@ -436,7 +375,6 @@ export class NotesView {
       icon.classList.add('fa-markdown-off');
       textarea.style.display = 'block';
       renderedContent.style.display = 'none';
-      toggleBtn.disabled = false;
       // Enable toolbar buttons
       toolbarButtons.forEach((button) => {
         button.disabled = false;
```

Comment:
instead of disabling, should we hide the markdown toolbar altogether? When we're previewing the markdown changes?

Reply:
Hmmm let me give that a shot

Reply:
Do you mean something like this? 
![image](https://github.com/user-attachments/assets/de65e387-19ee-4395-894d-32e2786f55f9)

Reply:
yeah! looks good